Tools/CMake/AMReX_Config.cmake

```
@@ -55,21 +55,25 @@ function (configure_amrex)
    # Setup OpenMP
    #
    if (ENABLE_OMP)
-      find_package(OpenMP REQUIRED CXX Fortran)
 
-      target_link_libraries(amrex
-         PUBLIC
-         OpenMP::OpenMP_CXX
-         OpenMP::OpenMP_Fortran)
+      find_package(OpenMP REQUIRED)
```

This probably works because the find module realizes that certain languages are already enabled and picks them up by default.

Just technically, I think this would probably still be better in an explicit form:
```suggestion
      find_package(OpenMP REQUIRED CXX)
```

If you want to avoid another `find_package(OpenMP)` for Fortran below, I would be tempted to write explicitly:
```suggestion
      if (CMAKE_Fortran_COMPILER_LOADED)
          find_package(OpenMP REQUIRED CXX Fortran)
      else()
          find_package(OpenMP REQUIRED CXX)
      endif()
```

---

I checked the `FindOpenMP.cmake` implementation: it seems that the check on `CMake_<LANG>_COMPILER_LOADED` is performed internally so I don't think it's necessary to re-do it on the amrex side

---

Thanks, that works for me.